services/docker-compose.yml

```diff
@@ -165,7 +163,7 @@ services:
   invitations:
     image: ${DOCKER_REGISTRY:-itisfoundation}/invitations:${DOCKER_IMAGE_TAG:-latest}
     init: true
-    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}"
```

Comment:
why no task.slot here? if it ever gets replicated (like when rolling out an update) it will conflict if by chance it starts on the same machine